azure-pipelines.yml

```
@@ -31,6 +31,14 @@ jobs:
     - script: git diff --exit-code --color
       displayName: Check tidy
 
+- job: CppLint
+  pool:
+    vmImage: 'ubuntu-latest'
+    container: sharaku/cpplint:latest
```

```suggestion
  container: sharaku/cpplint:latest
```

The container is not part of the pool. If you unindent this, it should work.